templates/image-builder.yml

```diff
@@ -63,6 +63,8 @@ objects:
             - name: composer-secrets
               mountPath: "/app/composer-secrets"
               readOnly: true
+            - name: config-volume
+              mountPath: /etc/config/image-builder
```

Comment:
The container doesn't follow FHS. To avoid any conflicts just put everything in `/app`. So `/app/quotas` or something should do.